web_dwc2.py

```diff
@@ -405,7 +413,7 @@ def rr_download(self, web_):
                 web_.write(f.read())
 
         else:
-            return {"err": 1}
+            return {"err": 1, "path": path_}
```

Comment:
ohehhhm. one sec. this is if athing is failed to report it to the webif, isnßt it. So you are reporting "error" with a valid content?

Reply:
ohh... debugging output...